<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TypesvalWidget</class>
 <widget class="QWidget" name="TypesvalWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>103</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ckeck file types</string>
  </property>
  <property name="windowIcon">
   <iconset theme="format-text-italic">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
   </property>
   <property name="rowWrapPolicy">
    <enum>QFormLayout::WrapLongRows</enum>
   </property>
   <property name="labelAlignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="formAlignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="actionLabel">
     <property name="text">
      <string>Action</string>
     </property>
     <property name="buddy">
      <cstring>actionCombo</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="actionCombo">
     <item>
      <property name="text">
       <string>Allow only selected extensions</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Deny selected extensions</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="extLabel">
     <property name="text">
      <string>Extensions</string>
     </property>
     <property name="buddy">
      <cstring>extEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="extEdit">
     <property name="placeholderText">
      <string>Comma separated extensions</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="levelLabel">
     <property name="text">
      <string>Level</string>
     </property>
     <property name="buddy">
      <cstring>levelSpin</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="levelSpin">
     <property name="specialValueText">
      <string comment="level">Any</string>
     </property>
     <property name="minimum">
      <number>-1</number>
     </property>
     <property name="maximum">
      <number>99</number>
     </property>
     <property name="value">
      <number>-1</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>actionCombo</tabstop>
  <tabstop>extEdit</tabstop>
  <tabstop>levelSpin</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
